### PR TITLE
Feature: Update  and Delete animals Table

### PR DESCRIPTION
**Pull request by oyelakinG9**
#  Update and Delete animals Table  🎉🎉

> Use database transactions.
> Modify and delete data in SQL.
> Prepare complex queries that answer analytical questions.
I added a column species of type string to the animal's table and modify the schema SQL file and a data SQL file.  Complex queries operation were performed for analytical questions.
The below screenshot shows the transactions.

Screenshot 1443-12-07 at 15 43 11

Screenshot 1443-12-07 at 15 44 27

Screenshot 1443-12-07 at 15 45 18

Screenshot 1443-12-07 at 15 46 19

Also performed the following queries operation shown in below screenshot 

Screenshot 1443-12-07 at 16 10 18

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -1,9 +1,11 @@
...
+  CREATE TABLE animals (
     id INT PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
     name VARCHAR NOT NULL,
     date_of_birth DATE,
     escape_attempts INT,
     neutered BOOLEAN,
-    weight_kg DECIMAL
-);
+    weight_kg DECIMAL,
+    species VARCHAR NOT NULL
```
Comment (SajjadAhmad14) [inline schema.sql]: Please user `ALTER TABLE` command to modify `animals` table.

File: queries.sql [hunk 2]
```diff
@@ -14,4 +14,50 @@ SELECT * FROM animals WHERE neutered=true;
...
 
-SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+SELECT * FROM animals;
+
+BEGIN;
+
+ALTER TABLE animals ADD species VARCHAR(50);
```
Comment (SajjadAhmad14) [inline queries.sql]: Please move this command to modify `animals` table from here to `schema.sql` file as everything that creates or change the tables structure should be saved in the `schema.sql` file.

File: queries.sql [hunk 3]
```diff
@@ -14,4 +14,50 @@ SELECT * FROM animals WHERE neutered=true;
...
+
+UPDATE animals SET species='unspecified';
+
+ROLLBACK;
```
Comment (SajjadAhmad14) [inline queries.sql:27]: Please verify that change was made before and after `Rollback` here.

File: queries.sql [hunk 4]
```diff
@@ -14,4 +14,50 @@ SELECT * FROM animals WHERE neutered=true;
...
+
+UPDATE animals SET species='pokemon' WHERE species IS NULL;
```
Comment (SajjadAhmad14) [inline queries.sql:33]: Please update the animals table by setting the species column to `digimon` for all animals that have a name ending in `mon`. Then commit the changes and kindly verify that change was made before and after `commit`.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Oyelaking, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
Git flow is followed. :heavy_check_mark:
README is professional. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### Please remember to tag me in your question so I can receive the notification.
**_Note: Please do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: schema.sql (hunk 1 above)
Comment (oyelakinG9) [reply schema.sql]: I used it but I forgot to add it the screenshot

File: queries.sql (hunk 2 above)
Comment (oyelakinG9) [reply queries.sql]: Thanks 😊
Modified

File: queries.sql (hunk 4 above)
Comment (oyelakinG9) [reply queries.sql:33]: Thanks.Man

All the changes were made. Although I forgot to add them to the queries SQL file. You can confirm them in the above screenshot. 👍🏻

**Review by hiromataba — APPROVED** [review]
# Status: Approved :tada: :tada: :tada: 
Hi @oyelakinG9    :wave: :wave: 

200w  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Oyelaking, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, whereby it appears already approved.


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by bimbolabuari — COMMENTED** [review]
Hello @oyelakinG9  👋,

_This is an additional review as a result of the second opinion request._

- Nice work implementing all the project requirement

Please ask for another review in your Dashboard and the next Code Reviewer will mark it as approve right away :)

Keep rocking 🚀!

**Review by omar-muhamad — APPROVED** [review]
# Approved ✔
### Hi @oyelakinG9,

Great job, Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

thumbs-up-agree


## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Github flow is used correctly. ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- N/A

## [Optional] suggestions:

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**